fastlane/Fastfile

```diff
@@ -133,6 +135,7 @@ platform :ios do
   lane :beta do
```

Comment:
I could be wrong, but I don't think this `beta` is unused. 

If that's the case, we could omit updating it, which would simplify the rest of the code by removing the need to extract the call to the Sentry action to upload dSYMs into its own lane.

We can then open another PR to remove the unused lane.